### DISCUSSION TITLE
Sources for Images

### DISCUSSION THREAD

**Discussion by newhinton** [nextcloud/unsplash#94] [Ideas]
## 👋 Welcome!
  This is a place to ask for more image sources! 
  If you have an imageprovider just like unsplash and want to see it here, post a comment!
  
  I would highly appreciate any help with that, so if you already know a link to such an api, post it aswell!

**Comment by newhinton** [comment]
Common Suggestions are:

Editing of parameters: eg. fireworks or nature (#63, #90, #24, #19, #4)

Images from a nextcloud folder (#6)

Other provider (#5)

**Reply by jet438** [reply]
I like all the suggestions above. I just wanted to ask if I could add pictures from my own collection of pictures versus the stock ones provided in the app.

**Reply by newhinton** [reply]
Generally i like the idea. However, this is not trivially to implement since i would need to write a whole management user interface for the images you want to display.

Just using a folder from your cloud will not be easily possible since that will not be available for every user. There was an idea with a shared link, but that adds its own problems. For now i will stick to more "conventional" sources, like unsplash in HD or wikimedia-commons. 

This does not mean it wont happen, just that i need more time to think this through.

**Reply by Bl4DEx** [reply]
Hi @newhinton 
I really like the idea of customizing your nextcloud instance with your own pictures. 

Is there any progress on your side since the last post? If not, is it feasable to use a folder within the nextcloud/apps/unsplash folder? I would think this should make permissions easier as php should have access to that folder path anyway.

On the UI under settings/splash/Image Provider, the admin could also select "local" from the already existing dropdown menu. Theming would go to default background, if the folder is empty (or pictures are not accessible due to OS permissions)

**Reply by AngelJRomero21** [reply]
Saludos, esto ha avanzado en algo?  me gustaria personalizar nextcloud
